code/game/gamemodes/gang/gang_things.dm

```diff
@@ -11,6 +11,10 @@
 	if(HAS_TRAIT(user, TRAIT_MINDSHIELD))
 		to_chat(user, "You attended a seminar on not signing up for a gang, and are not interested.")
 		return
+	var/datum/antagonist/gang/is_gangster = user.mind.has_antag_datum(/datum/antagonist/gang)
+	if(is_gangster.starter_gangster)
```

Comment:
runtimes if the person isn't a gangster which will be most of the time

Reply:
whoops, 1 sec